src/diffusers/pipelines/repaint/pipeline_repaint.py

```diff
@@ -13,33 +13,49 @@
 # limitations under the License.
 
 
-from typing import Optional, Tuple, Union
+from typing import List, Optional, Tuple, Union
 
 import numpy as np
 import torch
 
 import PIL
-from tqdm.auto import tqdm
 
 from ...models import UNet2DModel
 from ...pipeline_utils import DiffusionPipeline, ImagePipelineOutput
 from ...schedulers import RePaintScheduler
+from ...utils import deprecate
 
 
-def _preprocess_image(image: PIL.Image.Image):
-    image = np.array(image.convert("RGB"))
-    image = image[None].transpose(0, 3, 1, 2)
-    image = torch.from_numpy(image).to(dtype=torch.float32) / 127.5 - 1.0
+def _preprocess_image(image: Union[List, PIL.Image.Image, torch.Tensor]):
```

Comment:
Can we add a Copied from here from the img2img pipeline? so that it stays in sync?